Sources/FluentSQL/SQLSchemaConverter.swift

```diff
@@ -90,7 +90,7 @@ public struct SQLSchemaConverter {
                     name: SQLIdentifier(name)
                 )
             case .custom(let any):
-                return custom(any)
+                return SQLConstraint(algorithm: any as! SQLExpression, name: customName.map(SQLIdentifier.init(_:)))
```

Comment:
Is there any scenario where this would crash? Or would it be better to try the cast and fatal error with a suitable message if so?

Reply:
(Although I appreciate this is for fairly advanced users so people should know what they're doing)

Reply:
If this crashes, the user did something like `.constraint(.constraint(.custom("This is not an SQLExpression!"), name: "constraint")`, which is pretty blatantly wrong (and somewhat difficult to discover, since it involves explicitly invoking a case of `DatabaseQuery.Constraint` while still using the `.constraint()` helper on `QueryBuilder`). There are several other places where Fluent makes the force-unwrapped assumption that a `.custom()` is an `SQLExpression` once it gets to the `FluentSQL` layer (which is probably a pretty safe place for that assumption), so I'm not really worried about it.